experimental/e2e/storage/har.go

```diff
@@ -18,9 +18,64 @@ import (
 	"github.com/grafana/grafana-plugin-sdk-go/experimental/e2e/utils"
 )
 
+type file struct {
```

Comment:
I think it would be good to add a short comment about what these structs are for. Maybe even moving these to a separate `har_sync.go` for example to hide some of the deeper implementation details away?

Reply:
i moved `file` and `files` to `types.go` in [a04c90c](https://github.com/grafana/grafana-plugin-sdk-go/pull/491/commits/a04c90c45484a0c7046d177b47ad6f431045e072). it seemed like they are generic and could potentially be used by other file based storage implementations, but i'm happy to switch them if you'd prefer that they go in `har_sync.go`